addons/html_editor/static/tests/field.test.js

```diff
@@ -7,7 +7,7 @@ describe("monetary field", () => {
     test("should make a span inside a monetary field be unremovable", async () => {
         const content = unformat(`
             <p>
-                <span data-oe-model="product.template" data-oe-id="27" data-oe-field="list_price" data-oe-type="monetary" data-oe-expression="product.list_price" data-oe-xpath="/t[1]/div[1]/h3[2]/span[1]" class="o_editable">
+                <span data-oe-model="product.template" data-oe-id="27" data-oe-field="list_price" data-oe-type="monetary" data-oe-expression="product.list_price" data-oe-xpath="/t[1]/div[1]/h3[2]/span[1]" class="o_editable" contenteditable="true">
```

Comment:
Why did you add `contenteditable="true"` here? Shouldn't it be done automatically from `o_editable` ?

Reply:
@dmo-odoo Yes, you are right. `o_editable` will automatically set contentEditable true. However, in this test case, `contentBefore` and `contentAfter` take the same contents from the content variable. If I do not explicitly set `contentEditable=true` in the HTML of the content variable, the test case will fail because, by the time `contentAfter` is checked, `contentEditable=true` has already been added due to `o_editable`.

Now, I have modified the test case so that `contentBefore` and `contentAfter` take different contents.